src/main/webapp/script.js

```diff
@@ -10,7 +10,7 @@ function initBody() {
 }
 
 function checkPermissions() {
-  fetch('/check-permissions?projID='+config.projectID)
+  fetch('/check-permissions?projID='+configLogs.projectID)
```

Comment:
Have you changed the name of the config json structure in config? Just so I can make the same change if needed?

Reply:
Yes. I tried to solve _Uncaught SyntaxError: Identifier 'config' has already been declared_ by changing the name.